core/node/da_clients/Cargo.toml

```diff
@@ -55,3 +55,14 @@ pbjson-types.workspace = true
 
 # Eigen dependencies
 tokio-stream.workspace = true
+rlp.workspace = true
+kzgpad-rs = { git = "https://github.com/Layr-Labs/kzgpad-rs.git", tag = "v0.1.0" }
+rand.workspace = true
+sha3.workspace = true
+tiny-keccak.workspace = true
+alloy = { version = "0.3", features = ["full"] }
```

Comment:
we are trying to not use alloy in our codebase to avoid unnecessary dependencies, can we somehow replace it here?

Reply:
Fixed here  https://github.com/lambdaclass/zksync-era/pull/349